pettingzoo/butterfly/cooperative_pong/cooperative_pong.py

```diff
@@ -49,9 +49,9 @@ def get_valid_angle(randomizer):
 
     angle = 0
     while (
-        (angle > a1 and angle < b1)
-        or (angle > a2 and angle < b2)
-        or (angle > c1 and angle < d1)
+        (a1 < angle < b1)
+        or (a2 < angle < b2)
+        or (c1 < angle < d1)
         or (angle > c2)
```

Comment:
Heya, I don't think this works. It evaluates into `a1 < (angle < b1)`.

Reply:
Oh? It seems to be working in my test script. Am I missing something?

```python
def deg_to_rad(deg):
    return deg * np.pi / 180

ver_deg_range = 25
hor_deg_range = 10
a1 = deg_to_rad(90 - ver_deg_range)
b1 = deg_to_rad(90 + ver_deg_range)
a2 = deg_to_rad(270 - ver_deg_range)
b2 = deg_to_rad(270 + ver_deg_range)
c1 = deg_to_rad(180 - hor_deg_range)
d1 = deg_to_rad(180 + hor_deg_range)
c2 = deg_to_rad(360 - hor_deg_range)
d2 = deg_to_rad(0 + hor_deg_range)

randomizer, _ = seeding.np_random(seed=None)
for _ in range (100_000):
    angle = 2 * np.pi * randomizer.rand()

    x = [(angle > a1 and angle < b1),
         (angle > a2 and angle < b2),
         (angle > c1 and angle < d1),
         (angle > c2),
         (angle < d2),
         ]
    y = [(a1 < angle < b1),
         (a2 < angle < b2),
         (c1 < angle < d1),
         (angle > c2),
         (angle < d2),
         ]
    # r = ( a1 < (angle  < b1))
    # s = ((a1 <  angle) < b1)
    # print(f"{x[0]} {y[0]} {r} {s} {angle}")
    assert x == y, f"{x} {y} {angle}"

Reply:
Yeah that won't work, try this:

```py
print(True < 7)
print((5 < 6) < 7)
```

Reply:
Interesting. Curious who writes for Geeksforgeeks because [they have a whole page on it](https://www.geeksforgeeks.org/chaining-comparison-operators-python/), but I'll revert it back.

Reply:
Ok looking at the page, I actually never knew that... @pseudo-rnd-thoughts can you confirm?

Reply:
I'm not sure what python does under the hood I know that this works. 
Annoyingly you can't do this with numpy arrays